README.md

```diff
@@ -119,6 +119,27 @@ There are basically two methods to accomplish this:
   sha256 "efad782944da13d362aab9b81f001b7b8b1458794751de818e9848c47acd4b32"
   ```
 
+Installing GRPC on Unix
```

Comment:
Instead of recommending that the user clone the repo and update the submodules, I think we should instead recommend downloading the release package from the project's release page: https://github.com/grpc/grpc/releases.

e.g.

```
1. Download the 1.2 release: `curl -O https://github.com/grpc/grpc/archive/v1.2.0.tar.gz`
...
```

Even better is if we can suggest installation via a specific distro's package manager but I don't know how many versions of grpc core are available for Debian, etc.